### ISSUE TITLE
SSL certificate has expired

### ISSUE THREAD

**Issue by binarykitchen** [AveVlad/gulp-connect#140]
With `https` set to true, Google Chrome complains about the bad certificate.

It indeed has expired

```
Common Name (CN)	0.0.0.0
Organisation (O)	GruntJS
Organisational Unit (OU)	<Not Part Of Certificate>
Serial Number	00:E5:9B:8F:4A:5C:5E:39:83
...
Issued On	Thursday, 20 February 2014 at 12:15:44
Expires On	Friday, 20 February 2015 at 12:15:44
```

Can you update this? And probably automate this? Thx

**Comment by binarykitchen** [comment]
ping?

**Comment by binarykitchen** [comment]
heeellooo???

**Comment by binarykitchen** [comment]
heeeeeellloooo??

**Comment by ekarudianto** [comment]
Hi @binarykitchen I think I face the same problem as yours.. on activating the https, is there a way that gulp could read the certificates provided by our own ?

**Comment by wgcrouch** [comment]
You can tell it to use your own certificate, which is what I have done for now:

**Comment by ekarudianto** [comment]
thanks @wgcrouch you're a lifesaver !

**Comment by binarykitchen** [comment]
Well, then tell me why you still keep this outdated cert in the repo? Either update it or remove it

**Comment by justinmchase** [comment]
I'm not sure how to create the new ones here, @AveVlad or anyone.

If you lay out the steps I will update them.

**Comment by binarykitchen** [comment]
@justinmchase @AveVlad how to generate self signed ones? get some inspiration from this bash script i did:
https://github.com/binarykitchen/videomail-client/blob/master/bin/regenerate_ssl_certs.sh

**Comment by justinmchase** [comment]
I updated the certs using a slightly modified version of your script:
https://github.com/AveVlad/gulp-connect/blob/master/bin/make_certs.sh

I tested it locally and will publish an update. I can patch older versions too if anyone wants the certs updated for older versions.

**Comment by justinmchase** [comment]
Fixed in v3.2.2, by a6add2c591c10b17b4b05a7e6ddf40af99700bb2
